src/main/java/com/azure/cosmos/examples/crudquickstart/async/SampleCRUDQuickstartAsync.java

```diff
@@ -0,0 +1,342 @@
+            database = databaseResponse.getDatabase();
+            System.out.println("Checking database " + database.getId() + " completed!\n");
+            return Mono.empty();
+        }).block();
+        //  </CreateDatabaseIfNotExists>
+    }
+
+    private void createContainerIfNotExists() throws Exception {
+        System.out.println("Create container " + containerName + " if not exists.");
+
+        //  Create container if not exists
+        //  <CreateContainerIfNotExists>
+
+        CosmosContainerProperties containerProperties = new CosmosContainerProperties(containerName, "/lastName");
+        Mono<CosmosAsyncContainerResponse> containerIfNotExists = database.createContainerIfNotExists(containerProperties, 400);
+
+        //  Create container with 400 RU/s
+        containerIfNotExists.flatMap(containerResponse -> {
+            container = containerResponse.getContainer();
+            System.out.println("Checking container " + container.getId() + " completed!\n");
+            return Mono.empty();
+        }).block();
+
+        //  </CreateContainerIfNotExists>
+    }
+
+    private void createFamilies(Flux<Family> families) throws Exception {
+
+        //  <CreateItem>
+
+        final CountDownLatch completionLatch = new CountDownLatch(1);
+
+        //  Combine multiple item inserts, associated success println's, and a final aggregate stats println into one Reactive stream.
+        families.flatMap(family -> {
+                return container.createItem(family);
+            }) //Flux of item request responses
+            .flatMap(itemResponse -> {
+                System.out.println(String.format("Created item with request charge of %.2f within" +
+                    " duration %s",
+                    itemResponse.getRequestCharge(), itemResponse.getRequestLatency()));
+                System.out.println(String.format("Item ID: %s\n", itemResponse.getResource().getId()));
+                return Mono.just(itemResponse.getRequestCharge());
+            }) //Flux of request charges
+            .reduce(0.0,
+                (charge_n,charge_nplus1) -> charge_n + charge_nplus1
+            ) //Mono of total charge - there will be only one item in this stream
+            .subscribe(charge -> {
+                    System.out.println(String.format("Created items with total request charge of %.2f\n",
+                charge));
+            },
+                err -> {
+                    if (err instanceof CosmosClientException) {
+                        //Client-specific errors
+                        CosmosClientException cerr = (CosmosClientException)err;
+                        cerr.printStackTrace();
+                        System.err.println(String.format("Read Item failed with %s\n", cerr));
+                    } else {
+                        //General errors
+                        err.printStackTrace();
+                    }
+
+                    completionLatch.countDown();
```

Comment:
We should remove countdown latch for sample if possible, they are hard to understand.